moveit_ros/moveit_servo/config/servo_parameters.yaml

```diff
@@ -127,6 +98,16 @@ servo:
     }
 
 
+  incoming_command_timeout: {
+    type: double,
+    read_only: true,
```

Comment:
Same here. Seems this is super easy to tune at runtime given it's in a single condition